CMakeLists.txt

```diff
@@ -10,6 +10,13 @@ endif()
 #-g
 #endif()
 
+option(WITH_SSL "Turn OFF to compile without SSL" ON)
```

Comment:
The check below is redundant. OPTION() should take care of enabling/disabling the option.

Reply:
I don't think `option()` automatically adds a compile definition. The check is actually necessary.

Reply:
I stand corrected. Thank you.